src/Wallabag/CoreBundle/Controller/TagController.php

```diff
@@ -152,27 +152,40 @@ public function renameTagAction(Tag $tag, Request $request)
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            $entries = $this->get('wallabag_core.entry_repository')->findAllByTagId(
-                $this->getUser()->getId(),
-                $tag->getId()
-            );
-            foreach ($entries as $entry) {
-                $this->get('wallabag_core.tags_assigner')->assignTagsToEntry(
-                    $entry,
-                    $form->get('label')->getData()
+            $formLabel = $form->get('label')->getData();
+            $newTag = new Tag();
+            $newTag->setLabel($formLabel);
```

Comment:
This will avoid having to use a temporary variable

```suggestion
            $newTag->setLabel($form->get('label')->getData());
```